packages/dts-gen/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chevrotain/dts-gen",
```

Comment:
should we name it `cts-dts-gen` to be more precise?

Reply:
sounds good

Reply:
actually did you mean `cst-dts-gen`?

Reply:
yes, it was a typo 😄